mtp_bank_admin/apps/bank_admin/refund.py

```diff
@@ -35,5 +36,8 @@ def generate_refund_file(request):
     client = api_client.get_connection(request)
     client.bank_admin.transactions.patch(refunded_transactions)
 
+    # create file record
+    post_new_file(request, 'ACCESSPAY', [t['id'] for t in refunded_transactions])
```

Comment:
`'ACCESSPAY'` as a constant might be tidier especially if they're all in one place (be it this repo or common)?